spec/Section 5 -- Validation.md

```diff
@@ -135,6 +135,51 @@ extend type Dog {
 
 ## Operations
 
+### All Operation Definitions
+
+#### Operation Type Existence
+
+**Formal Specification**
+
+- For each operation definition {operation} in the document:
+  - Let {rootOperationType} be the _root operation type_ in {schema}
+    corresponding to the kind of {operation}.
+  - {rootOperationType} must exist.
```

Comment:
```suggestion
  - {rootOperationType} must be defined in the schema.
```

Reply:
We have existing algorithms that follow this pattern, e.g.

https://github.com/graphql/graphql-spec/blob/b41339a0fd22fb3ca9a1a6110334a7094f1f7643/spec/Section%205%20--%20Validation.md#L664-L666